src/components/List/VideoListStyled.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+const VideoListContainer = styled.div`
+  max-height: 200px;
+  margin: 20px auto;
+`;
+
+const VideoItemContainer = styled.div`
+  border-bottom: 1px solid #ccc;
+  display: block;
+  border-collapse: collapse;
+  width: auto;
+  height: auto;
+
+  & > img {
+    width: 30%;
+    padding: 1em;
+
+    @media screen and (max-width: 1000px) {
+      display: none;
+    }
+  }
+
+  & > a {
+    text-transform: uppercase;
+
+    & > p {
+      :hover {
```

Comment:
Instead of adding styling for a, p and img, you could add then at new different styled components like:L
```
const VideoAnchor = styled.a`
  text-transform: uppercase;
`;
```

Then using it at the JSX like:
```
<VideoItemContainer>
  <VideoInfo>
    <VideoAnchor />
  </VideoInfo>
</VideoItemContainer>
```